contracts/mockContracts/MockDGXDex.sol

```diff
@@ -0,0 +1,187 @@
+pragma solidity ^0.4.18;
+
+//ask:
+//dst address change?
+//should init another token outside in test?
+
```

Comment:
yes.
we have testToken contract for creating a token.
u can see it is done in many tests...
just init it and name it digix.

Reply:
done. removed the comments.